packages/app/src/server/models/bookmark-folder.ts

```diff
@@ -71,9 +84,32 @@ bookmarkFolderSchema.statics.findFolderAndChildren = async function(
     parentId?: Types.ObjectId | string,
 ): Promise<BookmarkFolderItems[]> {
```

Comment:
Also, `findFolderAndChildren` return type is `Promise<BookmarkFolderItems[] | null>`.
Please fix that.